### PR TITLE
PAYARA-4016 Mp Metrics does not work with multiple MBeans with the sa…

### PR DESCRIPTION
**Pull request by Cousjava**
…me name but different tags

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanMetadataHelper.java [hunk 1]
```diff
@@ -93,8 +94,13 @@
 
         List<MBeanMetadata> unresolvedMetadataList = resolveDynamicMetadata(metadataList);
         for (MBeanMetadata beanMetadata : metadataList) {
+            List<Tag> tags = new ArrayList<>();
+            for (fish.payara.microprofile.metrics.jmx.XmlTag tag : beanMetadata.getTags()) {
```
Comment (Pandrex247) [inline appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanMetadataHelper.java]: I'm aware it's moved from slightly further down, but is there a reason this class isn't imported? I don't see a conflict

**Comment by svendiedrichsen** [comment]
There are some Sonar-Bugs introduced just a few days ago to the class MBeanMetadata. You might want address these as well.

https://sonarcloud.io/project/issues?id=org.glassfish.main%3Aglassfish-main-aggregator&open=AWwWEaokTNTtgv3YW2fk&resolved=false&sinceLeakPeriod=true&types=BUG

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**